gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Feb 10 10:03:10 CET 2020
+#Sun Jan 24 19:04:26 CET 2021
```

Comment:
This is not change that we expect in commit. Please research how can we exclude this from automatic vcs commit changes.

Reply:
I changed it, yet I'm not sure how to deal with this problem.